### PR TITLE
Decouple UInt from non-serialization code

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

File: src/core/remote.js [hunk 1]
```diff
@@ -2065,8 +2065,7 @@ Remote.prototype.requestRippleBalance = function(options, callback) {
 
     // accountHigh implies for account: balance is negated.  highLimit is the
     // limit set by account.
-    const accountHigh = UInt160.from_json(options.account)
-    .equals(highLimit.issuer());
+    const accountHigh = options.account.equals(highLimit.issuer());
```
Comment (wltsmrz) [inline src/core/remote.js]: `options.account` is a ripple-lib/UInt160?

File: src/core/transaction.js [hunk 2]
```diff
@@ -1615,8 +1610,8 @@ Transaction.prototype.addMultiSigner = function(signer) {
   this.tx_json.Signers.push({Signer: signer});
 
   this.tx_json.Signers.sort((a, b) => {
-    return UInt160.from_json(a.Signer.Account)
-    .cmp(UInt160.from_json(b.Signer.Account));
+    return (new Buffer(decodeAddress(a.Signer.Account))).compare(
+      new Buffer(b.Signer.Account));
```
Comment (wltsmrz) [inline src/core/transaction.js]: Don't need to decodeAddress()?

File: test/remote-test.js [hunk 3]
```diff
@@ -4,16 +4,16 @@
...
 const Remote = require('ripple-lib').Remote;
 const Server = require('ripple-lib').Server;
 const Transaction = require('ripple-lib').Transaction;
-const UInt160 = require('ripple-lib').UInt160;
 const Currency = require('ripple-lib').Currency;
 const Amount = require('ripple-lib').Amount;
 const PathFind = require('ripple-lib')._test.PathFind;
 const Log = require('ripple-lib')._test.Log;
 
+const ACCOUNT_ONE = 'rrrrrrrrrrrrrrrrrrrrBZbvji';
```
Comment (wltsmrz) [inline test/remote-test.js]: Shouldn't we export constants from index/_test

File: src/core/remote.js (hunk 1 above)
Comment (clark800) [inline src/core/remote.js]: Good catch, should be string addresses: `options.account === highLimit.issuer()`

File: src/core/transaction.js (hunk 2 above)
Comment (clark800) [inline src/core/transaction.js]: Will fix

File: test/remote-test.js (hunk 3 above)
Comment (clark800) [inline test/remote-test.js]: sure

**Comment by clark800** [comment]
@wltsmrz updated

**Comment by wltsmrz** [comment]
LGTM

**Pull request merged**